imports/plugins/included/connectors-shopify/client/settings/shopify.js

```diff
@@ -143,7 +143,12 @@ Template.shopifySync.events({
 AutoForm.hooks({
   "shopify-connect-update-form": {
     onSuccess: function () {
-      return Alerts.toast(i18next.t("admin.settings.saveSuccess"), "success");
```

Comment:
my only thought here is that the return values are different now (previously: a Promise from Alerts; currently: `undefined`). does anything depend on this returning a promise?

Reply:
@pmn4 I don't think so because even before, that Promise from Alert wasn't being used anywhere. But let's also wait for smarter people(than me) to see this.

Reply:
@Akarshit I don't think @pmn4 means the promise from Alerts but the promise from the Stripe call.